tools/Azure.Mcp.Tools.EventGrid/src/Commands/Events/EventsPublishCommand.cs

```diff
@@ -38,13 +36,27 @@ public sealed class EventGridPublishCommand(ILogger<EventGridPublishCommand> log
         Secret = false
     };
 
+    private static readonly string[] s_item = ["cloudevents", "eventgrid", "custom"];
+
     protected override void RegisterOptions(Command command)
     {
         base.RegisterOptions(command);
         command.Options.Add(OptionDefinitions.Common.ResourceGroup);
         command.Options.Add(EventGridOptionDefinitions.TopicName);
         command.Options.Add(EventGridOptionDefinitions.EventData);
         command.Options.Add(EventGridOptionDefinitions.EventSchema);
+        command.Validators.Add(commandResult =>
+        {
+            var eventSchema = commandResult.GetValueOrDefault(EventGridOptionDefinitions.EventSchema);
+            if (!string.IsNullOrEmpty(eventSchema))
+            {
+                var normalizedSchema = eventSchema.Trim().ToLowerInvariant().Replace(" ", "");
```

Comment:
I was thinking about this and I feel like .Trim() could be removed because .Replace further along could do more than just the start and end.

Also, using hashset original.ignorecase string comparer, we don't have to change this to .ToLowerInvariant()

Reply:
Same as the last comment, will follow-up as this fix is blocking other PRs